src/main/resources/hudson/maven/Messages.properties

```diff
@@ -55,7 +55,7 @@ MavenProbeAction.DisplayName=Monitor Maven Process
 MavenProcessFactory.ClassWorldsNotFound=No classworlds*.jar found in {0} -- Is this a valid maven directory?
 
 MavenRedeployer.DisplayName=Deploy to Maven repository
-MavenVersionCallable.MavenHomeDoesntExist=Maven Home {0} doesn\u2019t exist
+MavenVersionCallable.MavenHomeDoesntExist=Maven Home {0} doesn't exist
```

Comment:
@daniel-beck I found that this would not load under Windows 8, which I traced to the use of the deprecated Unicode right-apostrophe character \u2019. Then I discovered that the ordinary ASCII apostrophe had been used elsewhere in this and other properties files, so it seemed logical to enforce consistency and fix the Windows build in one change! Hope this explains and justifies the change :-)